src/plots/cartesian/transition_axes.js

```diff
@@ -17,6 +17,8 @@ var Drawing = require('../../components/drawing');
 var Axes = require('./axes');
 var axisRegex = /((x|y)([2-9]|[1-9][0-9]+)?)axis$/;
 
+var LAST_TRANSLATION_RE = /translate\([^)]*\)\s*$/;
```

Comment:
Note: this will break or at least do something very unexpected if the last transform is not a vertical offset for the text. Not sure how to handle that more generally.